docs/getting-started/enterprise/key-connector.mdx

```diff
@@ -15,7 +15,7 @@ understand how it works.
 <Bitwarden>
 
 - A [local development server](../server/guide.md), running in the
-  [self-hosted configuration](../server/self-hosted/index.md)
+  [self-hosted configuration](../server/self-hosted/index.mdx)
```

Comment:
Unrelated fix for broken link
![Screenshot 2024-06-10 at 3 56 56 PM](https://github.com/bitwarden/contributing-docs/assets/1556494/a8e7d5d9-d871-4552-b769-c301830e09fa)